debezium-core/src/main/java/io/debezium/connector/common/BaseSourceTask.java

```diff
@@ -138,6 +138,20 @@ public final void start(Map<String, String> props) {
                     LOGGER.info("   {} = {}", propName, propValue);
                 });
             }
+            try {
+                this.coordinator = start(config);
+                setTaskState(State.RUNNING);
+            }
+            catch (Exception e) {
+                if (e instanceof RetriableException) {
```

Comment:
Could just catch the RetriableException and avoid the instanceof check.

Reply:
right, thanks for noting it! Fixed.